### PR TITLE
A little touch for betterC support

### PR DESCRIPTION
**Pull request by aferust**
A little touch for betterC support.

I needed this to run my game https://github.com/aferust/drawee

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ __dummy.html
 *.o
 *.obj
 *.a
+chipmunkd.lib
```
Comment (rcorre) [inline .gitignore]: lets just make this `*.lib`, no `.lib` files should be checked in. Also, you managed to commit your `chipmunkd.lib` anyways, could you remove that?

**Review by rcorre — COMMENTED** [review]
Thanks for the contribution! Could you remove `chipmunkd.lib`?

**Comment by rcorre** [comment]
Also, could you please update your commit message/messages be a little more detailed? https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html is a good guide.

**Comment by aferust** [comment]
I don't know how to edit "titles" of previous commit messages.

**Comment by aferust** [comment]
Do you want me to refork the repo and recommit the changes? I don't know any other way to change commit messages.

**Comment by rcorre** [comment]
Hi @aferust, that's actually all right. I'll just add details to the merge commit. Thanks!

**Comment by rcorre** [comment]
Thanks for taking the time to contribute this back!

**Comment by aferust** [comment]
Thank you! Could you please update the dub repo too?

**Comment by rcorre** [comment]
Released as `v1.0.1+7.0.1`, let me know if it doesn't show up in dub.

**Pull request merged**